loader/CMakeLists.txt

```diff
@@ -430,17 +430,13 @@ else()
     set_target_properties(vulkan PROPERTIES OUTPUT_NAME ${API_TYPE})
 
     if (LOADER_ENABLE_ADDRESS_SANITIZER)
-        target_compile_options(vulkan PUBLIC -fsanitize=address)
-        target_link_options(vulkan PUBLIC -fsanitize=address)
+        target_compile_options(vulkan PUBLIC -fsanitize=address,undefined)
+        target_link_options(vulkan PUBLIC -fsanitize=address,undefined)
```

Comment:
To save CI time just combine them.

Reply:
This affects regular developers (so, ME). Which is fine, I should be using this locally anyhow.